scripts/quests/windurst/Carbuncle_Debacle.lua

```diff
@@ -0,0 +1,193 @@
+------------------------------------
+-- Carbuncle Debacle
+-- !addquest 2 83
+-- Koru-moru !pos -120 -6 124 239
+--
+------------------------------------
+require('scripts/globals/items')
+require('scripts/globals/npc_util')
+require('scripts/globals/keyitems')
+require('scripts/globals/quests')
+require('scripts/globals/zone')
+require('scripts/globals/status')
+require('scripts/globals/interaction/quest')
+require('scripts/globals/titles')
+-----------------------------------------------
+local quest = Quest:new(xi.quest.log_id.WINDURST, xi.quest.id.windurst.CARBUNCLE_DEBACLE)
+--------------------------------------------------------------------------------------
+
+quest.reward =
+{
+    item            =   xi.items.EVOKERS_HORN,
+    fame            =   60,
+    fameArea        =   xi.quest.fame_area.WINDURST,
+    title           =   xi.title.PARAGON_OF_SUMMONER_EXCELLENCE
+}
+
+quest.sections =
+{
+    --Section: Quest Available
+
+    {
+
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+                player:getMainLvl() >= xi.settings.main.AF3_QUEST_LEVEL and
+                player:getMainJob() == xi.job.SMN and
+                not quest:getMustZone(player) and
+                player:getQuestStatus(xi.quest.log_id.WINDURST, xi.quest.id.windurst.CLASS_REUNION) == QUEST_COMPLETED
+        end,
+
+        [xi.zone.WINDURST_WALLS] =
+        {
+            ['_6n2'] = quest:progressEvent(415),
+
+            onEventFinish =
+            {
+                [415] = function(player, csid, option, npc)
+                    quest:begin(player)
+                    quest:setVar(player, 'Prog', 1)
+                end,
+            },
+        },
+    },
+
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED
+        end,
+
+        --Section: Quest Accepted
+        [xi.zone.WINDURST_WALLS] =
+        {
+            ['Koru-Moru'] =
+            {
+                onTrigger = function(player, npc)
+                    local questProgress = quest:getVar(player, 'Prog')
+
+                    if questProgress == 1 then
+                        return quest:progressEvent(416)
+                    elseif questProgress == 4 then
+                        return quest:progressEvent(417)
+                    elseif questProgress == 5 then
+                        return quest:event(418)
+                    elseif questProgress == 7 then
+                        return quest:progressEvent(419)
+                    elseif player:getQuestStatus(xi.quest.log_id.WINDURST, xi.quest.id.windurst.CARBUNCLE_DEBACLE) == QUEST_COMPLETED then
+                        return player:event(420)
```

Comment:
What are you trying to do here? This should be in the 

```lua 
check = function  (player, status, vars)
    return status == QUEST_ACCEPTED 
```
if it is change in the default dialog 

it would then be the above and then
```lua
['Koru-Moru'] = quest:event(420):replaceDefault(),
```

Reply:
This was a separate event that is suppose to happen after all 3 AF quests are completed

Reply:
Then yes it needs to go a quest == completed section

Reply:
there should be an example in 
https://github.com/AirSkyBoat/AirSkyBoat/blob/staging/scripts/quests/windurst/A_Pose_by_Any_Other_Name.lua#L185-L210